### PR TITLE
Update transactions.rb

### PR DESCRIPTION
**Pull request by raq929**
Explains how errors are propagated; gives an example of how to catch errors after ROLLBACK.

### CODE REVIEW

File: activerecord/lib/active_record/transactions.rb [hunk 1]
```diff
@@ -99,6 +99,16 @@ module Transactions
     # be propagated (after triggering the ROLLBACK), so you should be ready to
     # catch those in your application code.
     #
+    # Exceptions thrown by the ROLLBACK have a #message method that contains
+    # the ActiveRecord error message. 
+    #
```
Comment (arthurnn) [inline activerecord/lib/active_record/transactions.rb]: hey @raq929 thanks for the PR,
I wonder why do you mean by "Exceptions thrown by the ROLLBACK" , as far as I understand, for the example given, this error will come from somewhere else, in this case a validation error. That will eventually call a rollback, but the exception is not thrown by the rollback.

**Comment by raq929** [comment]
@arthurnn Good point! Updated the explanation.

File: activerecord/lib/active_record/transactions.rb [hunk 2]
```diff
@@ -99,6 +99,16 @@ module Transactions
     # be propagated (after triggering the ROLLBACK), so you should be ready to
     # catch those in your application code.
     #
+    # Exceptions that trigger the ROLLBACK have a #message method that contains
```
Comment (nateberkopec) [inline activerecord/lib/active_record/transactions.rb]: I wonder if we can rewrite this sentence to reflect that *all* Exceptions have a #message method.

Comment (nateberkopec) [inline activerecord/lib/active_record/transactions.rb]: "Exceptions that trigger the ROLLBACK will have a `message` containing the error(s) from ActiveRecord."

File: activerecord/lib/active_record/transactions.rb [hunk 3]
```diff
@@ -99,6 +99,16 @@ module Transactions
...
+    # the ActiveRecord error message. 
+    #
+    #   #Example error handling for an invalid record
```
Comment (nateberkopec) [inline activerecord/lib/active_record/transactions.rb]: I think your example code here is obvious, this comment is unnecessary.

**Comment by mikegee** [comment]
This is getting pretty old now, but it still seems valuable.

**Comment by rafaelfranca** [comment]
all exceptions has message so I don't think this is adding anything to the documentation. This is documenting exception handling in general and I don't thin the transaction documentation should be doing this. That said, I'm closing this PR.

Thank you so much for opening it.

**Pull request closed without merging**